src/upnp/client.js

```
@@ -0,0 +1,305 @@
+        break;
+      case 'stop':
+        await promisify(client, 'stop');
+        break;
+      case 'next':
+        // Not all renderers support next/previous
+        throw new Error('Next track not supported');
+      case 'previous':
+      case 'prev':
+        throw new Error('Previous track not supported');
+      case 'vol_abs':
+        const volume = Math.max(0, Math.min(100, Number(value) || 0));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)